### DISCUSSION TITLE
Thanos Support Policy

### DISCUSSION THREAD

**Discussion by shazib-summar** [thanos-io/thanos#6716] [Questions & Answers]
Hello, what is the support policy for Thanos. Which versions are supported at a given time? I went through the docs but couldn't find anything relevant to this.

**Comment by shazib-summar** [comment]
There is https://thanos.io/tip/thanos/security.md/#supported-versions but it seems very outdated. Is this information still up to date?

<img width="915" alt="image" src="https://github.com/thanos-io/thanos/assets/27294025/29e701f6-8dd4-4582-87de-e338d47986aa">

**Reply by yeya24** [reply]
Umm, yeah we should probably update this page in our doc.
There is not an explicit policy in Thanos about which versions are supported or not. But we always recommend using latest release as it comes with latest features and bug fixes.
